--- extra/dist/CMakeLists.txt	2016-09-02 04:17:15.095494747 +0300
+++ extra/dist/CMakeLists.txt	2016-09-02 04:28:12.165455806 +0300
@@ -47,7 +47,7 @@
 
 configure_file(default/tarantool.in default/tarantool @ONLY)
 install (FILES ${CMAKE_CURRENT_BINARY_DIR}/default/tarantool DESTINATION
-    ${CMAKE_INSTALL_SYSCONFDIR}/${SYSCONFIG_DEFAULT}/
+    ${CMAKE_INSTALL_FULL_SYSCONFDIR}/${SYSCONFIG_DEFAULT}/
     PERMISSIONS
     OWNER_READ OWNER_WRITE
     GROUP_READ
@@ -63,17 +63,17 @@
 
 # directories in /etc/ for tarantoolctl
 install(DIRECTORY DESTINATION
-    ${CMAKE_INSTALL_SYSCONFDIR}/${SYSCONFIG_AVAILABLEDIR}
+    ${CMAKE_INSTALL_FULL_SYSCONFDIR}/${SYSCONFIG_AVAILABLEDIR}
 )
 if (WITH_SYSVINIT)
     install(DIRECTORY DESTINATION
-        ${CMAKE_INSTALL_SYSCONFDIR}/${SYSCONFIG_ENABLEDDIR}
+        ${CMAKE_INSTALL_FULL_SYSCONFDIR}/${SYSCONFIG_ENABLEDDIR}
     )
 endif()
 
 # an example instance script for tarantoolctl
 install (FILES example.lua DESTINATION
-        ${CMAKE_INSTALL_SYSCONFDIR}/${SYSCONFIG_AVAILABLEDIR}
+	${CMAKE_INSTALL_FULL_SYSCONFDIR}/${SYSCONFIG_AVAILABLEDIR}
     PERMISSIONS
     OWNER_READ OWNER_WRITE OWNER_READ
     GROUP_READ WORLD_READ
@@ -83,13 +83,12 @@
 # Sic: chmod and chown are performed by rpm/deb
 install(DIRECTORY DESTINATION ${TARANTOOL_DATADIR})
 install(DIRECTORY DESTINATION ${TARANTOOL_LOGDIR})
-install(DIRECTORY DESTINATION ${TARANTOOL_RUNDIR})
 
 if (NOT TARGET_OS_FREEBSD)
 # logrotate files
 configure_file(tarantool.logrotate.in tarantool.logrotate @ONLY)
 install (FILES ${CMAKE_CURRENT_BINARY_DIR}/tarantool.logrotate
-    DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/logrotate.d/
+    DESTINATION ${CMAKE_INSTALL_FULL_SYSCONFDIR}/logrotate.d/
     RENAME "tarantool"
     PERMISSIONS
     OWNER_READ OWNER_WRITE
@@ -150,13 +149,3 @@
         GROUP_READ GROUP_READ
         WORLD_READ WORLD_READ)
 endif()
-
-if (WITH_SYSVINIT)
-    message (STATUS "Using scripts for sysvinit")
-    install (FILES tarantool.init DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/init.d/
-        RENAME tarantool
-        PERMISSIONS
-        OWNER_READ OWNER_WRITE
-        GROUP_READ GROUP_EXECUTE
-        WORLD_READ WORLD_EXECUTE)
-endif()
