diff --git a/magefile.go b/magefile.go
index d2f4816..890aced 100644
--- a/magefile.go
+++ b/magefile.go
@@ -214,7 +214,7 @@ func buildTt(argUpdaters ...optsUpdater) error {
 	mg.Deps(PatchCC)
 	mg.Deps(GenerateGoCode)
 
-	args := []string{"build", "-o", ttExecutableName}
+	args := []string{"build", "-o", ttExecutableName, "-mod", "vendor"}
 	var err error
 	for _, updateArguments := range argUpdaters {
 		if args, err = updateArguments(args); err != nil {
