diff --git a/debian/scripts/tarantool_logrotate b/debian/scripts/tarantool_logrotate
old mode 100755
new mode 100644
index 94c27e1..17a443f
--- a/debian/scripts/tarantool_logrotate
+++ b/debian/scripts/tarantool_logrotate
@@ -3,8 +3,8 @@
 use warnings;
 use strict;
 
-use constant CONFIG_DIR => '/var/lib/tarantool/started';
-use constant PID_DIR    => '/var/run/tarantool';
+use constant CONFIG_DIR => '/etc/tarantool';
+use constant PID_DIR    => '/run/tarantool';
 use File::Spec::Functions 'catfile';
 use File::Basename 'basename', 'dirname';
 use IO::Socket::INET;
@@ -13,7 +13,7 @@ exit 0 unless -x PID_DIR;
 exit 0 unless -x CONFIG_DIR;
 
 for (glob catfile PID_DIR, '*.pid') {
-    my $cfg = catfile CONFIG_DIR, basename $_, '.pid';
+    my $cfg = (catfile CONFIG_DIR, basename $_, '.pid') . '.cfg';
 
     unless(-r $cfg) {
         warn "Config file '$cfg' is not found\n";
diff --git a/debian/scripts/tarantool_snapshot_rotate b/debian/scripts/tarantool_snapshot_rotate
index 0fd2c29..f8fb7be 100644
--- a/debian/scripts/tarantool_snapshot_rotate
+++ b/debian/scripts/tarantool_snapshot_rotate
@@ -3,9 +3,9 @@
 use warnings;
 use strict;
 
-use constant CONFIG_DIR     => '/var/lib/tarantool/started';
-use constant PID_DIR        => '/var/run/tarantool';
-use constant SNAPSHOT_DIR   => '/var/lib/tarantool/snapshot';
+use constant CONFIG_DIR     => '/etc/tarantool';
+use constant PID_DIR        => '/run/tarantool';
+use constant SNAPSHOT_DIR   => '/var/lib/tarantool';
 use File::Spec::Functions 'catfile';
 use File::Basename 'basename', 'dirname';
 use IO::Socket::INET;
@@ -168,7 +168,7 @@ sub rotate_snapshot($$$) {
 
 DEBUGF "Looking through %s...", PID_DIR;
 for (glob catfile PID_DIR, '*.pid') {
-    my $cfg = catfile CONFIG_DIR, basename $_, '.pid';
+    my $cfg = (catfile CONFIG_DIR, basename $_, '.pid') . '.cfg';
 
     unless(-r $cfg) {
         warn "Config file '$cfg' is not found\n";
